net-dns/yaddns/yaddns-1.1.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A small dynamic DNS client"
+DESCRIPTION="Yaddns is a small dynamic dns client with multiple servers \
+support and a high flexibility."
+HOMEPAGE="https://yaddns.github.io/"
+COPYRIGHT="2015 Anthony Viallard and Raphael Huck"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/yaddns/yaddns-$portVersion.tar.gz"
```

Comment:
```diff
-SOURCE_URI="https://downloads.sourceforge.net/yaddns/yaddns-$portVersion.tar.gz"
+SOURCE_URI="https://github.com/yaddns/yaddns/archive/yaddns-$portVersion.tar.gz"
```
You forgot to switch back to GitHub. (The CHECKSUM_SHA256 is correct.)
What is confusing here is that both GitHub and SF use the same name for different tarballs.
BTW, after you change the tarball, you need to clean the workdir with `haikuporter -c yaddns`.

I think everything else is OK.